src/foam/nanos/so/pom.js

```diff
@@ -13,6 +13,14 @@ foam.POM({
       flags: "js|java" },
     { name: "SystemOutageTask",
       flags: "js|java" },
+    { name: "task/TaskInitPredicate",
```

Comment:
alphabetize please, so move all the task/ after the last System.
and if we are going with this extra directory, then SystemOutageTask should move there as well.

Reply:
will remove task directory

Reply:
done